Kickstarter-iOS/Views/Controllers/ShippingRulesTableViewController.swift

```diff
@@ -41,11 +41,19 @@ final class ShippingRulesTableViewController: UITableViewController {
         action: #selector(ShippingRulesTableViewController.dismissViewController)
       )
 
+    _ = self.navigationController?.navigationBar
+      ?|> \.barTintColor .~ UIColor.white
+      ?|> \.isTranslucent .~ false
+
+    _ = self.navigationController?.view
+      ?|> \.tintColor .~ UIColor.ksr_green_500
```

Comment:
I'm starting to thing we should set this globally in `AppDelegate` on the `window` object since we now have parts of the app using wrong tint color...thoughts: @justinswart @ifbarrera @Scollaco @cdolm92 ?